go/vt/vttablet/tabletserver/query_executor_test.go

```diff
@@ -288,13 +288,15 @@ func TestQueryExecutorPlanInsertSubQuery(t *testing.T) {
 	}
 }
 
+// RBR test is almost identical to the non-RBR test, except that
+// the _stream comments are suppressed for RBR.
 func TestQueryExecutorPlanInsertSubQueryRBR(t *testing.T) {
```

Comment:
I want to keep the comment for all the functions identical. Should I move it inside the functions instead?